<!-- This is a sample lanuch file, please change it based on your needs -->
<launch>
	
        <group ns="drone1">
    		<param name="tf_prefix" value="drone1" />
		<node name="RelocNode_tcp" pkg="RelocSensorDriver" type="RelocNode_tcp1.py" args="10.0.0.16 5000" />
		<node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" output="screen" clear_params="true" args="-ip 10.0.0.16"> 
	    		<param name="outdoor" value="0" />  <!-- This should disable wind speed estimation -->
	    		<param name="video_enable" value="0" />
	    		<param name="max_bitrate" value="1000" />
	    		<param name="bitrate" value="1000" />
	    		<param name="bitrate_ctrl_mode" value="0" />
	    		<param name="navdata_demo" value="1" />   <!-- This should reduce nv data publish speed to 15 Hz -->
	    		<param name="flight_without_shell" value="0" />
	   		<param name="altitude_max" value="3000" />
        		<param name="altitude_min" value="50" />
        		<param name="euler_angle_max" value="0.21" />
        		<param name="control_vz_max" value="700" />
	    		<param name="control_yaw" value="1.75" />
	    		<param name="detect_type" value="10" />
	    		<param name="enemy_colors" value="3" />
	    		<param name="detections_select_h" value="32" />
	    		<param name="detections_select_v_hsync" value="128" />
	    		<param name="enemy_without_shell" value="0" />
	    		<param name="do_imu_caliberation" value="false" />
	    		<param name="tf_prefix" value="mydrone" />
        		<!-- Covariance Values (3x3 matrices reshaped to 1x9)-->
        		<rosparam param="cov/imu_la">[0.1, 0.0, 0.0, 0.0, 0.1, 0.0, 0.0, 0.0, 0.1]</rosparam>
        		<rosparam param="cov/imu_av">[1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]</rosparam>
        		<rosparam param="cov/imu_or">[1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 100000.0]</rosparam>
		</node>
		<node name="drone_gui" pkg="tum_ardrone" type="drone_gui"/>
		<node name="drone_odometer" pkg="droneTracker" type="drone_odometry_publisher_mrs"/>
 	</group>

	<!-- Copy as necessary -->
	
	<!-- node name="rviz" pkg="rviz" type="rviz" args="-d $(find ariaClientDriver)/ariaClientDriver5.vcg" /-->
	
</launch>
